New package: DoubleBetaDecayGenerators v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: DoubleBetaDecayGenerators
- Repository: https://github.com/JuliaHEP/DoubleBetaDecayGenerators.jl
- Created by: @Yuan-Ru-Lin
- Version: v0.1.0
- Commit: 7a79ba1bcf476433c4ebc07eeb3192829d289d31
- Reviewed by: @Yuan-Ru-Lin
- Reference: https://github.com/JuliaHEP/DoubleBetaDecayGenerators.jl/issues/6#issuecomment-3256293803
- Description: Generators for ββ-decays
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## First release of DoubleBetaDecayGenerators.jl

One can instantiate double-beta decay data and draw random variates from them now.

```julia
data = TwoNuDBDData(:Ge76)
E1, E2, costheta = rand(data)
```
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598e2860aa901c285ac5d4d8547fe84f16298fb51f737920c4f59291d15d65378c2ceaad7fd9eebe1f26d778d480e3c3168216c31362d138046845a0d0b43757288f122649d3fa2ecfbae06c66085440563696cbf16c1871a7cfb0fae92807a67127b80dd7d0bb74e0b70911e3708c7bf0d33e0d7af3931891402be8cd182beb88006221cacce7aabc9f94ae306b59b339f -->

Yuan-Ru-Lin commented:
Let me publish the package once https://github.com/JuliaHEP/DoubleBetaDecayGenerators.jl/issues/1 is done.

JuliaRegistrator committed:
New package: DoubleBetaDecayGenerators v0.1.0

UUID: b67d9115-b479-4391-b710-1f027b87ba2b
Repo: https://github.com/JuliaHEP/DoubleBetaDecayGenerators.jl.git
Tree: 31af0c6e8163690db6cb8f620c323f80de3822eb

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
